script/application.go

```diff
@@ -119,13 +122,59 @@ func (a *Application) RenderProblems() string {
 	return strings.Join(problemStrings, "\n")
 }
 
-func (a *Application) GetData() string {
+func (a *Application) GetData() []byte {
 	data, err := json.MarshalIndent(a, "", "\t")
 	if err != nil {
 		log.Fatalf("Could not marshal Application data: %s", err.Error())
 	}
 
-	return string(data)
+	return data
+}
+
+// Take the application issue number and project name and turn it into
+// a file path. This will always be unique because it is relying on
+// github's issue numbers
+// e.g. 782-foo.json
+func (a *Application) FileName() string {
```

Comment:
[ignoreable] You might want some tests for this, if anything, to help demonstrate to future maintainers what kinds of inputs and outputs you expect.

Reply:
I think I'd like to punt on this in favor of more broadly refactoring and testing in a fast-follow.